### PR TITLE
rewrite all css in scss and use the webpack

### PR DESCRIPTION
**Pull request by kovpaka**

### CODE REVIEW

File: src/styles/_feature-list.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
+@use "./colors";
+.container--gray {
```
Comment (siarheikarabko) [inline src/styles/_feature-list.scss:2]: It's better to move `.container` classes to `common.scss`.

Comment (siarheikarabko) [reply src/styles/_feature-list.scss:2]: The same for `.icon` classes.

File: src/styles/_portfolio.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    margin-top: 50px;
+    background-color: colors.$color2;
+    color: colors.$color4;
+    &:hover,
+    &:focus {
+        background-color: colors.$color11;
+    }
+}
+
+%portfolio-border {
```
Comment (siarheikarabko) [inline src/styles/_portfolio.scss:85]: Should it be `.` ?

Comment (kovpaka) [reply src/styles/_portfolio.scss:85]: No?

**Pull request merged**